src/tests/ftest/deployment/critical_integration.py

```diff
@@ -174,7 +175,7 @@ def test_ras(self):
         # gather journalctl logs for each server host, verify system stop event was sent to logs
         results = get_journalctl(hosts=self.hostlist_servers, since=since,
                                  until=until, journalctl_type="daos_server")
-        str_to_match = "daos_engine exited: process exited with 0"
+        str_to_match = "daos_engine exited: process killed"
```

Comment:
In https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-15855/1/artifact/Functional%20Hardware%20Medium/deployment/critical_integration.py/job.log the journalctl log reports `/usr/bin/daos_engine exited: signal: killed`:
```
2025-02-07 23:44:16,458 stacktrace       L0045 ERROR| Feb 07 23:43:25 wolf-143.wolf.hpdd.intel.com daos_server[100101]: instance 1, pid 101354, rank 2 exited with status: /usr/bin/daos_engine exited: signal: killed
...
2025-02-07 23:44:16,460 stacktrace       L0045 ERROR| Feb 07 23:43:35 wolf-143.wolf.hpdd.intel.com daos_server[100101]: instance 0, pid 101567, rank 5 exited with status: /usr/bin/daos_engine exited: signal: killed
```

Reply:
Ugh. Thanks